DigitalOffice/Tests/TimeManagementServiceTests/Mappers/WorkTimeMapperTests.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using LT.DigitalOffice.TimeManagementService.Database.Entities;
+using LT.DigitalOffice.TimeManagementService.Mappers;
+using LT.DigitalOffice.TimeManagementService.Mappers.Interfaces;
+using LT.DigitalOffice.TimeManagementService.Models;
+using LT.DigitalOffice.TimeManagementServiceUnitTests.UnitTestLibrary;
+using NUnit.Framework;
+
+namespace LT.DigitalOffice.TimeManagementServiceUnitTests.Mappers
+{
+    public class WorkTimeMapperTests
+    {
+        private readonly IMapper<CreateWorkTimeRequest, DbWorkTime> mapper = new WorkTimeMapper();
+
+        private CreateWorkTimeRequest request;
+        private DbWorkTime expectedDbWorkTimeWithoutId;
+
+        [SetUp]
+        public void SetUp()
+        {
+            request = new CreateWorkTimeRequest
+            {
+                ProjectId = Guid.NewGuid(),
+                StartTime = new DateTime(2020, 7, 29, 9, 0, 0),
+                EndTime = new DateTime(2020, 7, 29, 9, 0, 0),
+                Title = "I was working on a very important task",
+                Description = "I was asleep. I love sleep. I hope I get paid for this.",
+                WorkerUserId = Guid.NewGuid()
+            };
+
+            expectedDbWorkTimeWithoutId = new DbWorkTime
+            {
+                ProjectId = request.ProjectId,
+                StartTime = request.StartTime,
+                EndTime = request.EndTime,
+                Title = request.Title,
+                Description = request.Description,
+                WorkerUserId = request.WorkerUserId
+            };
+        }
+
+        [Test]
+        public void ThrowExceptionIfArgumentIsNull()
+        {
+            Assert.Throws<ArgumentNullException>(() => mapper.Map(null));
+        }
+
+        [Test]
+        public void ReturnRightModelSuccessfully()
+        {
+            var newWortTime = mapper.Map(request);
+            expectedDbWorkTimeWithoutId.Id = newWortTime.Id;
+
+            Assert.IsInstanceOf<Guid>(newWortTime.Id);
+            SerializerAssert.AreEqual(expectedDbWorkTimeWithoutId, newWortTime);
+        }
+
+        [Test]
+        public void ReturnRightModelWithNullDescriptionSuccessfully()
+        {
```

Comment:
Is this a CheckRights or TimeManagement service? haha xD

gotcha, Pavel.
TBH I also messed this up when copypasting some content from ProjectService